tests/useMultiRootEditor.test.tsx

```diff
@@ -9,8 +9,8 @@ import { describe, beforeEach, afterEach, it, expect, vi } from 'vitest';
 import React, { useEffect } from 'react';
 import { CKEditorError } from 'ckeditor5';
 import { render, waitFor, renderHook, act } from '@testing-library/react';
-import useMultiRootEditor, { EditorEditable, EditorToolbarWrapper } from '../src/useMultiRootEditor.tsx';
-import { ContextWatchdogContext } from '../src/context/ckeditorcontext.tsx';
+import useMultiRootEditor, { EditorEditable, EditorToolbarWrapper } from '../src/useMultiRootEditor';
+import { ContextWatchdogContext } from '../src/context/ckeditorcontext';
 import { timeout } from './_utils/timeout.js';
 import { createDefer } from './_utils/defer.js';
 import { createTestMultiRootWatchdog, TestMultiRootEditor } from './_utils/multirooteditor.js';
```

Comment:
Those imports have extensions, and the other don't. I wonder if it's some convention or if we could unify it.
```suggestion
import { timeout } from './_utils/timeout';
import { createDefer } from './_utils/defer';
import { createTestMultiRootWatchdog, TestMultiRootEditor } from './_utils/multirooteditor';
```